dbt_platform_helper/commands/copilot.py

```diff
@@ -168,6 +168,9 @@ def _normalise_keys(source: dict):
             normalised_environments[env] = _normalise_keys(initial)
 
         for env_name, env_config in environments.items():
+            env_config = (
+                {} if env_config is None else env_config
+            )  # ensures an env config can be nullable
```

Comment:
Would this be more obvious and not need the comment as...
```suggestion
            if env_config is None:
                env_config = {}
```